syntax = "proto3";

message RetrieveLocation {
    int32 id = 1;
}

message LocationMessage {
    optional int32 id = 1;
    int32 person_id = 2;
    string creation_time = 3;
    string latitude = 4;
    string longitude = 5;
}


service LocationService {
    rpc Get(RetrieveLocation) returns (LocationMessage);
    rpc Create(LocationMessage) returns (LocationMessage);
}
